js/stat_test.ts

```diff
@@ -8,7 +8,7 @@ testPerm({ read: true }, async function statSyncSuccess(): Promise<void> {
   assert(packageInfo.isFile());
   assert(!packageInfo.isSymlink());
 
-  const testingInfo = Deno.statSync("testing");
+  const testingInfo = Deno.statSync("node_modules");
   assert(testingInfo.isDirectory());
   assert(!testingInfo.isSymlink());
```

Comment:
It looks like this assertion should fail. After all, 'node_modules' is a symlink. So I wonder why it doesn't.

Reply:
lstat vs stat

Reply:
Nevermind. stat vs lstat.